src/integrationTest/java/uk/gov/hmcts/darts/arm/client/ArmApiClientIntTest.java

```diff
@@ -86,10 +90,25 @@ void updateMetadataShouldSucceedIfServerReturns200Success() {
                                           .and(matchingJsonPath("$.itemId", equalTo(externalRecordId)))
                                   ));
 
-        assertEquals(OK, updateMetadataResponse.getStatusCode());
-        var body = updateMetadataResponse.getBody();
-        assertEquals(UUID.fromString(externalRecordId), body.getItemId());
-        assertFalse(body.isError());
+        assertEquals(UUID.fromString(externalRecordId), updateMetadataResponse.getItemId());
     }
 
+    @Test
+    void downloadArmDataShouldSucceedIfServerReturns200Success() {
+        // Given
+        byte[] serverResponse = "some binary content".getBytes();
+        stubFor(
+            WireMock.get(urlPathMatching(DOWNLOAD_ARM_DATA_PATH))
+                .willReturn(
+                    aResponse()
+                        .withHeader("Content-type", "application/octet-stream")
+                        .withBody(serverResponse)
+                        .withStatus(200)));
+
+        // When
+        byte[] response = armApiClient.downloadArmData("bearer token", CABINET_ID, EXTERNAL_RECORD_ID, EXTERNAL_FILE_ID);
```

Comment:
Should the first param here be "Bearer token"?

Reply:
if we want to be precise yes. I will change this 👍